pkg/rancher-desktop/backend/images/nerdctlImageProcessor.ts

```diff
@@ -67,11 +67,18 @@ export default class NerdctlImageProcessor extends imageProcessor.ImageProcessor
       false);
   }
 
-  async scanImage(taggedImageName: string): Promise<imageProcessor.childResultType> {
+  async scanImage(taggedImageName: string, namespace: string): Promise<imageProcessor.childResultType> {
+    // environment variables and the trivy command name are included in the command, which may be invoked like:
+    // limactl shell 0 sudo CONTAINERD_ADDRESS=… trivy …
     return await this.runTrivyCommand(
       [
+        'CONTAINERD_ADDRESS=/run/k3s/containerd/containerd.sock',
```

Comment:
Has this been tested on Windows? You're not going through `sudo` (nor `env`) there.